pysyncobj/utility.py

```diff
@@ -0,0 +1,77 @@
+import os
+import time
+
+from .encryptor import getEncryptor
+from .node import Node, TCPNode
+from .poller import createPoller
+from .tcp_connection import TcpConnection
+
+
+class TcpUtility(object):
+
+    def __init__(self, password=None, timeout=900.0):
+        self.__timeout = timeout
+        self.__poller = createPoller('auto')
+        self.__connection = TcpConnection(self.__poller,
+                                          onDisconnected=self.__onDisconnected,
+                                          onMessageReceived=self.__onMessageReceived,
+                                          onConnected=self.__onConnected,
+                                          timeout=timeout)
+        if password is not None:
+            self.__connection.encryptor = getEncryptor(password)
+
+        self.__result = None
+        self.__error = None
+
+    def sendMessage(self, node, message):
```

Comment:
I suggest to rename it into 'executeCommand' and make it return result / error. Otherwise it's not obvious why sendMessage run polling loop inside and wait for disconnect.